### PR TITLE
Immovable rods penetrate harder and show ghosts how many clongs they've racked up

### PR DESCRIPTION
**Pull request by Ryll-Ryll**
## About The Pull Request
Come in late to a looping immovable rod and want to know the score? Has it racked up so many bodies that no one even knows anymore? Rods now keep track of how many mobs, sentient mobs, and sentient carbons they've penetrated, which ghosts can view by examining the rod. 

[](https://i.imgur.com/5yOAu6U.png)

Also, rods can now cause piercing wounds to the chest, since you're getting the shit penetrated out of you anyway. Total damage should still be 160, armor and limb percentage notwithstanding.

## Why It's Good For The Game
Provides an accurate count of how much mayhem a rod has caused, and blasting through people with the wounds makes rods more fun to watch. Keeping track with the rest of deadchat might be a bit less engaging since you can just look and see the exact number, but I think watching the numbers climb higher and higher will be fun in its own way.

## Changelog
:cl: Ryll/Shaps
add: Observers can examine immovable rods to see how many mobs/sentient mobs/sentient carbons it has penetrated
tweak: Immovable rods can now cause piercing wounds to the chest
/:cl:

### CODE REVIEW

File: code/modules/events/immovable_rod.dm [hunk 1]
```diff
@@ -137,13 +158,23 @@ In my current plan for it, 'solid' will be defined as anything with density == 1
...
-	if(ishuman(L))
-		var/mob/living/carbon/human/H = L
-		H.adjustBruteLoss(160)
-	if(L && (L.density || prob(10)))
-		L.ex_act(EXPLODE_HEAVY)
+/obj/effect/immovablerod/proc/penetrate(mob/living/smeared_mob)
+	smeared_mob.visible_message("<span class='danger'>[smeared_mob] is penetrated by an immovable rod!</span>" , "<span class='userdanger'>The rod penetrates you!</span>" , "<span class='danger'>You hear a CLANG!</span>")
+
+	num_mobs_hit++
+	if(smeared_mob.client)
```
Comment (ATH1909) [inline code/modules/events/immovable_rod.dm:165]: don't make it count dead mobs

**Comment by Tlaltecuhtli** [comment]
make it play combo sounds

**Comment by TheVekter** [comment]
Hey, while you're in there, can you see if you can figure out why the rod gets stuck on certain objects? The only ones I know off the top of my head are the gateway and the dressers in the dorms on Delta.

**Review by Rohesie — APPROVED** [review]
very silly

**Pull request merged**